core/src/main/java/com/yahoo/ycsb/Client.java

```diff
@@ -1,802 +1,802 @@
-/**                                                                                                                                                                                
```

Comment:
Please don't include formatting changes as it makes reviewing the substantive changes much harder.